src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs

```diff
@@ -185,6 +185,12 @@ public void SourceInAnotherClassPassingSomeDataToConstructor(int n, int d, int q
             Assert.AreEqual(q, n / d);
         }
 
+        [Test, Category("Top"), TestCaseSource("HereIsTheDataWithParametersFor1765", new object[] { 8000, 8, 1000 })]
+        public void SourceInAnotherClassPassingSomeDataToConstructor1765(int n, int d, int q)
```

Comment:
The whole point of the issue is to allow use of parameters with data in the same class, specifying only the method name. You've implemented that correctly, but the name of your test says you are testing data in another class. Also, use of the issue number in the test name won't help anyone reviewing the code after we merge it. If you just call it SourceInSameClassPassingSomeDataToConstructor, that tells someone reading the code next year what the test does.